test/uptech_growthbook_sdk_flutter_test.dart

```diff
@@ -10,6 +10,13 @@ class ToglTest extends UptechGrowthBookWrapper {
 
 void main() {
   const String featureName = 'some-feature-name';
+  const String greaterThan = '\$gt';
```

Comment:
please move these inline into the respective tests.

This makes it so that all the necessary information to evaluate a test exists within the locality of the test.

Tests are one of the few places where you need to be less concerns with being dry.

When there is a need to be dry in tests generally it should be done not by moving things up to the top of the file or parenting group but instead into shared utility functions for the test suite.